packages/grpc-js/src/server.ts

```diff
@@ -970,12 +983,69 @@ export class Server {
         this.channelzTrace.addTrace('CT_INFO', 'Connection established by client ' + clientAddress);
         this.sessionChildrenTracker.refChild(channelzRef);
       }
+      let connectionAgeTimer: NodeJS.Timer | null = null;
+      let connectionAgeGraceTimer: NodeJS.Timer | null = null;
+      let sessionClosedByServer = false;
+      if (this.maxConnectionAgeMs !== UNLIMITED_CONNECTION_AGE_MS) {
+        // Apply a random jitter within a +/-10% range
+        const jitterMagnitude = this.maxConnectionAgeMs / 10;
```

Comment:
Is this formula part of a protocol? Or just a one-off heuristic?

Reply:
Ah, nevermind. I see it inside of A9.

Reply:
It's in the spec I linked:

> A per-connection random jitter of +/-10% will be added to MAX_CONNECTION_AGE to spread out connection storms.